<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlibcHeapBinsDialog</class>
 <widget class="QDialog" name="GlibcHeapBinsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableView" name="viewBins"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Chain info:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="chainInfoEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Detailed chunk info:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="placeholderText">
          <string>Enter chunk base address and press enter</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
